tested/languages/javascript/templates/run.mako

```diff
@@ -91,12 +91,11 @@ try {
         ${ctx.before}
         % for testcase in ctx.testcases:
             writeSeparator();
+            <% testcase: _TestcaseArguments %>\
```

Comment:
Why was this moved outside the `try-catch`?

Reply:
Because when you declare a variable in the `try-catch` it'sn't visible in the other `try-catches`